shimmy/utils/meltingpot.py

```diff
@@ -0,0 +1,61 @@
+"""Shared utils for meltingpot."""
```

Comment:
This looks rather similar to the dm_env utils, do you think the two could be merged, especially the spec_to_space functionality? Or perhaps rename the timestep_to_observations into multiagent_timestep_to_observations to make it more verbose.

Reply:
Good call I’ll look into that